dialog/base_test.go

```diff
@@ -0,0 +1,29 @@
+package dialog_test
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/dialog"
+	"fyne.io/fyne/internal/painter/software"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/test"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+
+	"testing"
+)
+
+func TestDialog_Background_Group(t *testing.T) {
```

Comment:
I know this came from background being suspicious, but in this instance I don't know if the test actually cares about background so the name is a little misleading.

Should we leave the content as is? just an open question - it doesn't matter much one way or another.